### PR TITLE
Add textarea component

### PR DESCRIPTION
**Pull request by ZeProf2Code**
- Added the textarea component

### CODE REVIEW

File: resources/views/components/textarea.blade.php [hunk 1]
```diff
@@ -0,0 +1,3 @@
+@props(['disabled' => false])
+
+<textarea {{ $disabled ? 'disabled' : '' }} {!! $attributes->merge(['class' => 'form-input rounded-md shadow-sm']) !!}></textarea>
```
Comment (bonzai) [inline resources/views/components/textarea.blade.php:3]:
```suggestion
<textarea {{ $disabled ? 'disabled' : '' }} {!! $attributes->merge(['class' => 'form-textarea rounded-md shadow-sm']) !!}></textarea>
```

Comment (bonzai) [reply resources/views/components/textarea.blade.php]: See #353.

**Comment by taylorotwell** [comment]
You can add this in your own project.

**Comment by srdjanmarjanovic** [comment]
Actually, i find it useful to have textarea along with the input

**Comment by GLStephen** [comment]
@taylorotwell do you mind providing any context for why you would reject textarea from the project? Jetstream has every other input type, buttons, labels, sections, modals, etc. 

Why not just put the last remaining HTML form element into the project instead of sending users on a vision quest to find it? Users will inevitably build their first, second or third form and need a textarea and start pawing through code only to find it doesn't exist.

It's not like other relative esoterica in the form world isn't already there. It's just, basically, this one missing form element. Care to explain?

**Comment by troccoli** [comment]
I don't really understand either why this PR was closed, but thank you @ZeProf2Code. I was able to use your code and add a textarea Jetstream component to my project.

**Pull request closed without merging**